backend/src/main/java/com/woowacourse/naepyeon/controller/MessageController.java

```diff
@@ -26,35 +28,40 @@ public class MessageController {
     private final MessageService messageService;
 
     @PostMapping
-    public ResponseEntity<CreateResponse> createMessage(@RequestBody @Valid final MessageRequest messageRequest,
+    public ResponseEntity<CreateResponse> createMessage(@AuthenticationPrincipal LoginMemberRequest loginMemberRequest,
+                                                        @RequestBody@Valid final MessageRequest messageRequest,
                                                         @PathVariable final Long rollingpaperId) {
-        final Long messageId = messageService.saveMessage(messageRequest.getContent(), messageRequest.getAuthorId(),
+        final Long messageId = messageService.saveMessage(messageRequest.getContent(), loginMemberRequest.getId(),
                 rollingpaperId);
```

Comment:
사소한거긴 한데, 아래와 같은 컨벤션은 어떤가요?
```suggestion
        final Long messageId = 
                messageService.saveMessage(messageRequest.getContent(), loginMemberRequest.getId(), rollingpaperId);
```

Reply:
해당 부분 반영했습니당!